Jenkinsfile_CNP

```diff
@@ -148,10 +160,14 @@ withPipeline(type, product, component) {
       sh """
       eval \$(./bin/variables/load-preview-environment-variables.sh ${CHANGE_ID})
       ./bin/pull-latest-camunda-files.sh ${camundaBranch}
+      ./bin/pull-latest-dmn-files.sh ${dmnBranch}
+      ./bin/pull-latest-camunda-wa-files.sh ${waStandaloneBranch}
     """
+      //uploadDmnDiagrams('preview',"${CHANGE_ID}")
```

Comment:
The function has been added on line `58` but it's only usage would be here and this is commented out.

Is this intentional?

Reply:
Looks like the script is called in `pull-latest-dmn-files.sh`, so this commend can be removed and so can the related function.